src/driver/odoc_unit.ml

```diff
@@ -1,15 +1,31 @@
 type pkg_args = {
   pages : (string * Fpath.t) list;
   libs : (string * Fpath.t) list;
+  pages_linked : (string * Fpath.t) list;
```

Comment:
To make the 1-1 correspondence clear, could this be `pages : (string * ~odoc:Fpath.t * ~odocl:Fpath.t) list` ?

Reply:
Yeah, it's a bit poor as it stands currently. Let me see if it can be improved.

Reply:
Perfect :)